samples/sensor/lm73c/README.rst

```diff
@@ -0,0 +1,43 @@
+.. _lm73c:
+
+lm73c Temperature Sensor
+###################################
+
+Overview
+********
+
```

Comment:
Here a link explaining what an LM73C temperature sensor is and where more information can be found would be helpful.

Reply:
Just me being lazy and not looking up how to add a link in to a .rst file. I shall do so.